package.json

```diff
@@ -83,6 +83,6 @@
     "shady-css-parser": "0.0.8",
     "split": "^1.0.0",
     "strip-indent": "^2.0.0",
-    "typescript": "^2.1.4"
+    "typescript": "2.1.4"
```

Comment:
I'd prefer to just change libraryCache's type to `Map<string, string|undefined>`, because we want Typescript 2.2's ability to analyze mixins.